website/pages/docs/drivers/docker.mdx

```diff
@@ -629,16 +629,16 @@ through Nomad plugins or dynamic job configuration.
 
 ## Capabilities
 
-The `docker` driver implements the following [capabilities](/docs/internals/plugins/task-drivers#capabilities-capabilities-error)
+The `docker` driver implements the following [capabilities](/docs/internals/plugins/task-drivers#capabilities-capabilities-error).
 
 | Feature | Implementation |
 | --- | --- |
-| SendSignals | true |
-| Exec | true |
-| FSIsolation | image |
-| NetIsolationModes | host, group, task |
-| MustInitiateNetwork | true |
-| MountConfigs | all |
+| `nomad alloc signal` | true |
+| `nomad alloc exec` | true |
+| filesystem isolation | image |
+| network isolation | host, group, task |
+| create / destroy network namespace | true |
```

Comment:
Hm so I'm realizing this one probably isn't useful to document as it's purely an internal implementation detail. There's a generic implementation of create/destroy network that's used if the driver does not implement it.

I'd hate for people to assume our `exec` driver doesn't fully support network namespaces because it has `false` for this value.

Reply:
Oh yeah, that makes sense if we think of this as "operator documentation", which was the whole point of having the plain English info here. Will fix.

Reply:
Fixed!